app/views/mailers/notifications_mailer/nudge_to_request_mentoring.html.haml

```diff
@@ -0,0 +1,10 @@
+%p Hi #{@user.handle},
+%p You've completed a few exercises (nice work!) but not yet requested mentoring on any of your solutions.
```

Comment:
At first I thought: "a few exercise", but then I realized that students will all have done `hello-world` :) (although you can't request mentoring on that)

Reply:
Yes - I wondered if we should hold off and only do it when someone has done three exercises, but it made the query too heavy :)